libs/ledger-live-common/src/load/speculos.ts

```diff
@@ -159,20 +161,10 @@ export async function createSpeculosDevice(
     ...(sdk ? ["--sdk", sdk] : []),
     "--display",
     "headless",
-    "--vnc-password",
-    "live",
+    ...(process.env.CI ? ["--vnc-password", "live", "--vnc-port", "41000"] : []),
     ...(isSpeculosWebsocket
-      ? [
-          "--apdu-port",
-          "40000",
-          "--vnc-port",
-          "41000",
-          "--button-port",
-          "42000",
-          "--automation-port",
-          "43000",
-        ]
-      : ["--api-port", "40000", "--vnc-port", "41000"]),
+      ? ["--apdu-port", "40000", "--button-port", "42000", "--automation-port", "43000"]
+      : ["--api-port", "40000"]),
```

Comment:
Should come from `getEnv("SPECULOS_API_PORT")` no ?

Reply:
The port is passed [here](https://github.com/LedgerHQ/ledger-live/blob/7bbad43beab706a98b03ff9147cc67f289220c44/libs/ledger-live-common/src/load/speculos.ts#L141);
It is determined by calculation with the `SPECULOS_PID_OFFSET` env and then stored in the `SPECULOS_API_PORT` to use for communication between LLD and Speculos